float64 motor_angle1
float64 motor_angle2
float64 motor_angle3
float64 motor_angle4
float64 x
float64 y
float64 z
uint8 	pump
uint8 	swiftpro_status
uint8 	gripper

